jax/numpy/lax_numpy.py

```diff
@@ -1816,6 +1816,10 @@ def asarray(a, dtype=None, order=None):
   lax._check_user_dtype_supported(dtype, "asarray")
   return array(a, dtype=dtype, copy=False, order=order)
 
+@_wraps(onp.copy)
+def copy(a, order='K'):
+  return array(a, copy=True, order=order)
```

Comment:
Looking at our implementation of `array`, it basically ignores the `copy` argument. But I guess it doesn't hurt to pass it for consistency? 🤷‍♂

Reply:
That was my thinking: reduce it to an existing policy!